### ISSUE TITLE
I've runned your project,But I cannot get the results that your claimed.could you give me some tips?

### ISSUE THREAD

**Issue by gogo03** [JunYeopLee/fast-autoaugment-efficientnet-pytorch#5]
resnet34 results can be shown:
python train.py --seed=24 --scale=5 --optimizer=sgd --fast_auto_augment=True
.........
[+] Training step: 62000/64000	Training epoch: 0/351	Elapsed time: 318.71min	Learning rate: 0.0011729701340847298
  Acc@1 : 88.281%
  Acc@5 : 99.219%
  Loss : 0.3259652554988861
  FW Time : 104.842ms
  BW Time : 152.699ms

[+] Valid results
  Acc@1 : 91.840%
  Acc@5 : 99.880%
  Loss : 1.2543833255767822

I also test resnet20.
with fast-automentation:   valid  Acc@1:92.18%
without fast-automentaion valid:Acc@1:92.2%
Pre-policies..........................................................92.4%

and I download found-policies,It has 80 subpolices.
But I use you project,It only generated 8 subpolices to random choice.
the generated 8 subpolices is like the following.
RandomChoice(
    Compose(
    Pad(padding=4, fill=0, padding_mode=constant)
    RandomCrop(size=(32, 32), padding=None)
    RandomHorizontalFlip(p=0.5)
    Contrast(prob=0.76, magnitude=0.20)
    ShearXY(prob=0.91, magnitude=0.54)
    ToTensor()
)
    Compose(
    Pad(padding=4, fill=0, padding_mode=constant)
    RandomCrop(size=(32, 32), padding=None)
    RandomHorizontalFlip(p=0.5)
    Contrast(prob=0.75, magnitude=0.54)
    Posterize(prob=0.70, magnitude=0.63)
    ToTensor()
)
    Compose(
    Pad(padding=4, fill=0, padding_mode=constant)
    RandomCrop(size=(32, 32), padding=None)
    RandomHorizontalFlip(p=0.5)
    Sharpness(prob=0.19, magnitude=0.90)
    Brightness(prob=0.22, magnitude=0.92)
    ToTensor()
)
    Compose(
    Pad(padding=4, fill=0, padding_mode=constant)
    RandomCrop(size=(32, 32), padding=None)
    RandomHorizontalFlip(p=0.5)
    Brightness(prob=0.09, magnitude=0.28)
    TranslateXY(prob=0.38, magnitude=0.16)
    ToTensor()
)
    Compose(
    Pad(padding=4, fill=0, padding_mode=constant)
    RandomCrop(size=(32, 32), padding=None)
    RandomHorizontalFlip(p=0.5)
    Contrast(prob=0.29, magnitude=0.31)
    Color(prob=0.57, magnitude=0.39)
    ToTensor()
)
    Compose(
    Pad(padding=4, fill=0, padding_mode=constant)
    RandomCrop(size=(32, 32), padding=None)
    RandomHorizontalFlip(p=0.5)
    Color(prob=0.64, magnitude=0.66)
    Sharpness(prob=0.69, magnitude=0.87)
    ToTensor()
)
    Compose(
    Pad(padding=4, fill=0, padding_mode=constant)
    RandomCrop(size=(32, 32), padding=None)
    RandomHorizontalFlip(p=0.5)
    Equalize(prob=0.48, magnitude=0.84)
    Brightness(prob=0.76, magnitude=0.00)
    ToTensor()
)
    Compose(
    Pad(padding=4, fill=0, padding_mode=constant)
    RandomCrop(size=(32, 32), padding=None)
    RandomHorizontalFlip(p=0.5)
    Equalize(prob=0.33, magnitude=0.21)
    AutoContrast(prob=0.50, magnitude=0.86)
    ToTensor()
)
)
